include/pistache/stream.h

```diff
@@ -83,10 +83,10 @@ namespace Pistache
     public:
         using Base = StreamBuf<CharT>;
 
-        explicit ArrayStreamBuf(size_t maxSize)
+        explicit ArrayStreamBuf(size_t _maxSize)
             : StreamBuf<CharT>()
             , bytes()
-            , maxSize(maxSize)
+            , maxSize(_maxSize)
```

Comment:
Were these leading underscores added by mistake?

Reply:
Intentionally added to suppress a compiler warning about the argument parameter shadowing a class attribute by the same name.